packages/jest-resolve/src/isBuiltinModule.ts

```diff
@@ -7,10 +7,14 @@
 
 import module = require('module');
 
-// "private" api
-declare const process: {
-  binding(type: string): {};
-};
+declare global {
+  namespace NodeJS {
+    export interface Process {
+      // "private" api
+      binding(type: string): {};
```

Comment:
won't this pollute the global definition of all consumers as well? I'm down with adding `pnp` as optional, but this one is purposefully not added to global types by DT and is highly discouraged to use by Node itself

Reply:
Yeah I realised this after I pushed. I'll restore the declare and just add `& process`.

Reply:
fwiw this *should* be "fine" as it only takes effect when this file is imported, which none of the `d.ts`s do since it's not a public function, but I'll change it for peace of mind all the same.